canonical-websites/www.ubuntu.com
# V1 cloud overview

## Done

Update /cloud to vbt including new foundation cloud section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud](http://0.0.0.0:8001/cloud))
- Compare to the [design](https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-overview.png)
- Demo www.ubuntu.com-v1-cloud-setup.demo.haus

## Issue / Card

Foundation cloud: https://trello.com/c/tuTC0Ucn

### Docs

Cloud overview: https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#
